Fix some typos and use consistent names

pauamma opened the pull request:
(The French composer is Edouard Lalo. Eduardo Lalo is the Puerto Rican writer.)

pauamma committed:
Fix some typos and use consistent names

(The French composer is Edouard Lalo. Eduardo Lalo is the Puerto Rican writer.)

pauamma commented:
Are you (the repo committer(s)) waiting for something from me?

pauamma commented:
Ping @stesser @lsalvadore

lsalvadore commented on calendar.music:
```diff
@@ -107,14 +107,14 @@
 03/08	Hector Berlioz dies in Paris, 1869
 03/08	Carl Philipp Emanuel Bach is born, 1714
-03/08	Carlo Gesuldo is born (date uncertain), 1566
+03/08	Carlo Gesualdo is born (date uncertain), 1566
```
Because of the uncertainty, I would just drop this line. I have done only a superficial research, but according to what I have found the uncertainty regards both the year and the day of the month. Only thing certain enough seems to be the month.

lsalvadore commented on calendar.music:
```diff
@@ -470,7 +470,7 @@
 11/24	Scott Joplin born near Linden, Texas, 1867
 11/25	"The Last Waltz" concert is played by The Band at Winterland, 1976
-11/25	Johann Strauss, Jr., writes 'On the Beautiful Blue Danube', 1867
+11/25	Johann Strauss II, writes 'On the Beautiful Blue Danube', 1867
```
After a quick research I have found around both "Johann Strauss Jr." and "Johann Strauss II" (and also "the Younger", "the Son"). Why do you recommend using "Johann Strauss II"?

If we were to keep the Jr version, I would remove the comma between "Strauss" and "Jr.".

bsdimp replied:
Chicago manual of style has recommended a comma before Jr, but not before II or III, etc. I learned that in school in the 80s. However, since its 93 edition, the CMA has stopped recommending that. The New Yorker style guide, however, continues to recommend a comma. MLA suggests using it. The APA recommends a comma. I'm not sure which style guide we follow.

lsalvadore replied:
Thanks, I did not know about this.

According to `grep -R "Jr"`, at the moment we have 4 occurences of Jr with a comma and 1 without it, so I would say that keeping the comma is more consistent. However, since we have only 5 occurences of Jr, we can also easily change the repository style, if we wanted. In any case it would be nice to adopt the same style in all cases.

pauamma replied:
> After a quick research I have found around both "Johann Strauss Jr." and "Johann Strauss II" (and also "the Younger", "the Son"). Why do you recommend using "Johann Strauss II"?

Familiarity and because I never came across Jr. But you sent me digging, and searching for "Johann Strauss II,Johann Strauss Sohn,Johann Strauss Jr,Johann Strauss Junior" in Google Ngram's German corpus only shows Sohn used at all, so I would use "Son" on second thought. With or without a comma I can't say. And I don't think we use any style guide in the sense of adopting substantial chunks of it. Since these run to hundreds of pages (over 1000 for the Chicago Manual of Style IIRC), I'm fine with that.+

pauamma committed:
Address review comments

While there, fix more spelling

pauamma commented:
Keyboard is acting up again. I'll remove spurious + crap ASAP.

pauamma committed:
Remove spurious characters my keyboard spewed

While there, fix a missing cap and add diacritics.

pauamma commented:
I've started adding diacritics to the French names, because that's what I can do without looking up reference works, but the ones that need it most are the Slavic names. Before I embark into that, are there reasons not to use any?

lsalvadore commented:
You should be able to use any diacritic safely as long as it is is supported by UTF-8 thanks to commit 61c8e36fe7e83cdbabe2d265bdaeba7c3531d2b5. Do you want to add diacritics now for this pull request or is this pull request ready for merging?

pauamma commented:
> You should be able to use any diacritic safely as long as it is is supported by UTF-8 thanks to commit [61c8e36](https://github.com/freebsd/calendar-data/commit/61c8e36fe7e83cdbabe2d265bdaeba7c3531d2b5). Do you want to add diacritics now for this pull request or is this pull request ready for merging?

Probably best to add them here, for consistency.

pauamma committed:
Add all (I think) missing diacritics.

While there, fix a few missing words and inconsistencies in names of people and places.

pauamma commented:
> You should be able to use any diacritic safely as long as it is is supported by UTF-8 thanks to commit [61c8e36](https://github.com/freebsd/calendar-data/commit/61c8e36fe7e83cdbabe2d265bdaeba7c3531d2b5). Do you want to add diacritics now for this pull request or is this pull request ready for merging?

Ready for merging (or at least pre-merging review) now.

lsalvadore commented:
Merged, thanks.
